### PR TITLE
402-refactor: day.js impotrs refactored

### LINKED ISSUE #402: Refactor Day.js imports
## 📝 Describe the refactoring task

Move `day.js` config to `shared` slice.
Then refactor all `day.js` imports to use this `dayJS` re-export

Image

The goal is to have only one import of public api for day.js library, that is used across the app, in order to not confuse new developers.

## 📍 Code location

Image

### PR DESCRIPTION
**Pull request by ismoilkhon**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [x] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description
Day.js configs moved to a shared slice and all imports that relate to day.js are refactored.

- Related Issue #402 
- Closes #402 

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Summary by CodeRabbit

- **Chores**
  - Updated import paths for `dayJS` across multiple files for consistency and maintainability.

### CODE REVIEW

File: src/shared/helpers/getActualDataList.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { dayJS } from '@/app/services/dayjs';
+import { dayJS } from '../dayjs';
```
Comment (Quiddlee) [inline src/shared/helpers/getActualDataList.ts:1]:
```suggestion
import { dayJS } from '@/shared/helpers/dayJS';
```

File: src/shared/dayjs/index.ts
Comment (Quiddlee) [inline src/shared/dayjs/index.ts:1]: Please, move `dayJS` to helpers

image

**Comment by Quiddlee** [comment]
Please change related tickets section according to this:

image

File: src/shared/dayjs/index.ts
Comment (ismoilkhon) [reply src/shared/dayjs/index.ts:1]: Ok

File: src/shared/helpers/getActualDataList.ts (hunk 1 above)
Comment (Wystov) [reply src/shared/helpers/getActualDataList.ts]: but as @natanchik mentioned, we need this service only for tests, and should use lib in code by direct import 
```
import dayjs from 'dayjs';
```

Comment (Quiddlee) [reply src/shared/helpers/getActualDataList.ts]: At the last meeting, we decided to use only one specific import to avoid confusion

**Review by Quiddlee — APPROVED** [review]

**Review by Wystov — APPROVED** [review]

**Pull request merged**